src/actions/guides/database/migrations.cr

```diff
@@ -155,9 +157,18 @@ class Guides::Database::Migrations < GuideAction
     ```crystal
     alter :users do
       add active : Bool, default: true, fill_existing_with: true
-      add otp_code : String, fill_existing_with: my_custom_otp_code_generator
+      add otp_code : String, fill_existing_with: "fake-otp-code-123"
     end
     ```
+    
+    > You can also use `fill_existing_with: :nothing` if your table is empty.
+    
+    If a static value will not work, try this:
+    
+    * If you have not yet released the app, consider using `fill_existing_with: :nothing`
+      and dropping the database `lucky db.drop` and recreating it `lucky db.create && lucky db.migrate`.
+    * Consider making the type nilable `add otp_code : String?`, then fill the values with whatever value you need.
+      Then later make it required with `make_required :otp_code`
```

Comment:
Where would `make_required :otp_code` go? Is that a model method?

Reply:
It's available in migrations. I'll add an example!